core/domain/stats_jobs_one_off.py

```diff
@@ -303,6 +314,22 @@ def reduce(exp_id, values):
 
     @classmethod
     def prepare_reduce(cls, exp_id, values):
+        """Prepares the reduce for different statistic models of the given
```

Comment:
Runs the reduce step to extract aggregate exploration statistics from the dicts passed in by the mapping function.